Cargo.toml

```diff
@@ -194,6 +194,9 @@ members = [
 	"utils/wasm-builder",
 ]
 
+[patch.crates-io]
+dusk-plonk = { git = "https://github.com/maticnetwork/plonk.git", tag = "v0.8.2-polygon-1" }
```

Comment:
What is the key change in this fork as compared to the original dusk-plonk?

Reply:
Basically these changes are just to **make "public" some functions, types, and members** we are using in our code. You can see the diff between `0.8.2` and `polygon/0.8.2` [here](https://github.com/maticnetwork/plonk/compare/v0.8.2...polygon/0.8.2)